libraries/classes/Controllers/View/CreateController.php

```diff
@@ -92,70 +92,83 @@ public function __invoke(ServerRequest $request): void
         ];
 
         // View name is a compulsory field
-        if (isset($_POST['view']['name']) && empty($_POST['view']['name'])) {
+
+        /** @var array|null $view */
+        $view = $request->getParsedBodyParam('view');
+
+        if (isset($view['name']) && empty($view['name'])) {
```

Comment:
```suggestion
        if (empty($view['name'])) {
```
Redundant check can be removed while you are at it.